### PR TITLE
Home

### PR DESCRIPTION
**Pull request by mohammed-sas**
Screen Shot 2022-02-02 at 6 46 11 PM

### CODE REVIEW

File: Home/home.css [hunk 1]
```diff
@@ -30,6 +30,25 @@ main::-webkit-scrollbar {
...
+    display: flex;
+    gap: 1rem;
+}
+.featured-item {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    border: 1px solid var(--purple-700);
+    border-radius: 1rem;
+    background: #fff;
```
Comment (Ayushi20-19) [inline Home/home.css]: very minor you can use var here also.

**Review by Ayushi20-19 — APPROVED** [review]

File: Home/home.html [hunk 2]
```diff
@@ -60,7 +61,25 @@ <h4 class="grey">Racquets</h4>
                     <h1>Badminton</h1>
                 </div>
             </div>
-
+            <h2 class="centered-text grey">Featured Categories</h2>
+            <div class="featured-categories">
+                <div class="featured-item">
+                    <img src="../Images/rq-3.png" alt="">
```
Comment (VishalPatil18) [inline Home/home.html]: It's good practice to give value to the `alt` attribute as it helps in accessibility. You could give value to this image and all the following one's.

File: Home/home.css (hunk 1 above)
Comment (mohammed-sas) [reply Home/home.css]: thank you for the suggestion

**Review by Sudhanshusoni45 — APPROVED** [review]
Looks Good to go!

**Review by VishalPatil18 — APPROVED** [review]
Just one suggestion was added. The rest looks quite cool! Great stuff @mohammed-sas 👍 

You're a quite fast man!

File: Home/home.html (hunk 2 above)
Comment (mohammed-sas) [reply Home/home.html]: sure. thank you for the compliments

**Pull request merged**